experimental/python/databricks/bundles/jobs/_models/cron_schedule.py

```diff
@@ -19,14 +19,14 @@ class CronSchedule:
     A Cron expression using Quartz syntax that describes the schedule for a job. See [Cron Trigger](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html) for details. This field is required.
     """
 
-    pause_status: VariableOrOptional[PauseStatus] = None
+    timezone_id: VariableOr[str]
     """
-    Indicate whether this schedule is paused or not.
+    A Java timezone ID. The schedule for a job is resolved with respect to this timezone. See [Java TimeZone](https://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html) for details. This field is required.
     """
 
-    timezone_id: VariableOr[str] = "UTC"
+    pause_status: VariableOrOptional[PauseStatus] = None
```

Comment:
Curious -- what triggered reordering?

Reply:
required fields must come first. Previously the field was not required because it had a default value.